src/pages/Merchant/LoginPage/LoginPage.js

```diff
@@ -37,9 +41,18 @@ function LoginPage() {
 
   const handleFormSubmit = (event) => {
     event.preventDefault();
-    if (!validateForm()) {
-      return;
-    }
+    if (!validateForm()) return;
+
+    const loginDetails = {
+      mobile: "",
+      isMerchant: true,
+      password: passRef.current.value,
+      email: emailRef.current.value,
+    };
+    loginMerchant(loginDetails).then(async (res) => {
+      if (!res) return;
+      console.log("logged in");
```

Comment:
show a toast.success("Logged in successfully")

user should know he is successfully logged in